kcidb/monitor/output.py

```diff
@@ -184,9 +187,12 @@ def render(self):
             subject = subject[:NOTIFICATION_MESSAGE_SUBJECT_MAX_LEN - 2] + "✂️"
             LOGGER.warning("Subject is %s characters too long, truncated",
                            subject_extra_characters)
-        assert NOTIFICATION_MESSAGE_SUBJECT_RE.fullmatch(subject), \
-            f"Subject is invalid, must match " \
-            f"{NOTIFICATION_MESSAGE_SUBJECT_RE.pattern} regex"
+        if not NOTIFICATION_MESSAGE_SUBJECT_RE.fullmatch(subject):
+            subject = re.sub(f"[{NOTIFICATION_MESSAGE_SUBJECT_INVALID_CHARS}]",
+                             "�", subject)
```

Comment:
Ah, wait, this is the "replacement character". We need to use the "uncertainty sign" character instead: `⯑`. The former indicates something wrong with Unicode encoding, and would be confusing.